README.md

```diff
@@ -30,145 +30,144 @@ Styleguide for low-fidelity paper prototyping and wireframing.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/33.PNG)
 
-3.4. _MUST_. **Images** must be represented by a square with an x mark touching its corners.
+3.4. _MUST_ represent **images** by a square with an x mark touching its corners (`☒`).
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/34.PNG)
 
-3.5. _SHOULD_. For **tables** and **general containers** that includes repeatable elements, draw only the first line of element and include three vertical dots at the middle of container.
+3.5. _SHOULD_ draw only the first line of a repeating elements, followed by a vertical ellipsis (`⋮`) at the middle of  the container for **tables** and **containers**.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/35.PNG)
 
-3.6. _SHOULD_. For **tables** and **general containers** that is supposed to be taller than wireframe bounds, DO NOT finish the container with a bottom side line, and apply a three vertcial dashes following each horizontal side of container.
+3.6. _SHOULD NOT_ draw bottom lines for **tables** and **containers** that are taller than the wireframe. Instead, apply three dashes (like an ellipsis) from each vertical line of the container's sides.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/36.PNG)
 
-3.7. _SHOULD_. For **tables** and **general containers** that is supposed to be wider than wireframe bounds, DO NOT finish the container with a bottom side line, and apply a three horizontal dashes following each vertical side of container.
+3.7. _SHOULD NOT_ draw right/left lines for **tables** and **containers** that are wider than the wireframe. Instead, apply three dashes (like an ellipsis) from each horizontal line of container's sides.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/37.PNG)
 
 ### 4. Blue elements 💙 Features
 
-4.1. _MUST_. Use the **blue** color to describe the content, origin and destination of features.
+4.1. _MUST_ use **blue** color to describe features' content, origin and destination.
 
-4.2. _MUST_. Define a **reference code** for each feature with a letter, starting from `A` to `Z`.
+4.2. _MUST_ define **reference codes** for each feature with a letter, starting from `A` to `Z`.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/42.PNG)
 
-4.3. _SHOULD NOT_. Prefer omiting control labels and instead use the reference code described in 4.2.
+4.3. _SHOULD NOT_ label control elements directly. Instead, prefer using the reference code described in 4.2.
 
-4.4. _MUST_. Build a "**Table of Features**" including the reference code and the main details of each in a usable free space of the wireframe. If there is no free space in the wireframe, use the back of the sheet to make it.
+4.4. _MUST_ make a "**features table**" including each component reference code and their main details. Use any free space in the wireframe. If there isn't, use the back of the sheet to make it.
```

Comment:
`Table of Features` was meant to be like a `Table of Contents`, a.k.a. `ToC`, present in a bunch of documentations and books.

Reply:
I wasn't aware of this. I'm changing back.
However, a quick google showed me that is also usually written as "Contents".

"A table of contents, usually headed simply "Contents" and abbreviated informally as TOC"

Reply:
https://en.wikipedia.org/wiki/Table_of_contents